kms/key/Acornfile

```diff
@@ -25,6 +26,8 @@ args: {
 	pendingWindowDays: 30
 	// AWS IAM policy to attach to the Key. Optional.
 	keyPolicy: {}
+	// Removal policy for the key. Valid options are "DESTROY" (key will be scheduled for deletion when this acorn is deleted) and "RETAIN" (key will remain in AWS even after this acorn is deleted). Default is "DESTROY".
+	removalPolicy: "DESTROY"
```

Comment:
How long does it take for the deletion to processed after scheduling?

Reply:
Anywhere from 7 to 30 days with no way to bypass it. The user can set this number with the `pendingWindowDays` argument.